#diagnostic
string DeviceName
string Node_Name
uint8 System
uint8 SubSystem
uint8 Component
uint8 Diagnostic_Type
uint8 Level
uint8 Diagnostic_Message
string Description
